domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/pathAssertions.kt

```diff
@@ -20,6 +21,9 @@ fun <T : Path> _exists(assertionContainer: Expect<T>): Assertion =
 fun <T : Path> _existsNot(assertionContainer: Expect<T>): Assertion =
     ExpectImpl.builder.createDescriptive(assertionContainer, NOT_TO, RawString.create(EXIST)) { it.notExists }
 
+fun <T : Path> _endsWith(assertionContainer: Expect<T>, other: T): Assertion =
+    ExpectImpl.builder.createDescriptive(assertionContainer, TO, RawString.create(ENDS_WITH)) { it.endsWith(other) }
```

Comment:
that's not what you want. Make a test fail on purpose and have a look how it looks like

Reply:
Thanks. I changed the descriptive to match with requirement